docs/code-snippets/office-snippets.yaml

```
@@ -3958,51 +3958,81 @@ Office.EventType:enum:
             });
         });
     }
-Office.File#getSliceAsync:member(1):
+Office.File:interface:
   - |-
-    // This sample shows how to get all the slices of a file. 
-    // The asynchronous operation returns a Promise so it can be awaited.
-    private getAllSlices(file: any): Promise<any> {
-        const self = this;
-        let isError = false;
-
-        return new Promise(async (resolve, reject) => {
-            let documentFileData = [];
-            for (let sliceIndex = 0; (sliceIndex < file.sliceCount) && !isError; sliceIndex++) {
-                const sliceReadPromise = new Promise((sliceResolve, sliceReject) => {
-                    file.getSliceAsync(sliceIndex, (asyncResult) => {
-                        if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
-                            documentFileData = documentFileData.concat(asyncResult.value.data);
-                            sliceResolve({
-                                IsSuccess: true,
-                                Data: documentFileData
-                            });
-                        } else {
-                            file.closeAsync();
-                            sliceReject({
-                                IsSuccess: false,
-                                ErrorMessage: `Error in reading the slice: ${sliceIndex} of the document`
-                            });
-                        }
-                    });
-                });
-                await sliceReadPromise.catch((error) => {
-                    isError = true;
-                });
-            }
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/refs/heads/main/samples/excel/26-document/get-file-in-slices-async.yaml
```

For consistency with similar links in snippets.yaml

```suggestion
    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/26-document/get-file-in-slices-async.yaml
```